### DISCUSSION TITLE
How does gindex-v4 manage to add subs using .SRT files?

### DISCUSSION THREAD

**Discussion by atmos-o** [tks18/gindex-v4#237] [Q&A]
As far as I knew adding subtitles for web was only possible with .VTT files. How is this project managing to do so?

**Comment by tks18** [comment]
I created a inline tool to convert srt to vtt inside the website which will automatically convert srt to vtt whenever it detects there is a srt file

**Comment by atmos-o** [comment]
Does this process this client side or is done through JSDelivr?

**Comment by tks18** [comment]
Client side
